sqlite_query.c

```diff
@@ -239,7 +239,11 @@ sqlite_convert_to_pg(Form_pg_attribute att, sqlite3_value * val, AttInMetadata *
 					{
```

Comment:
Please indent for case block code with 1 tab (including braces)

Reply:
I have added additional tab, @MinhLA1410. Really here was no normal indent in my code. Please verify.

Reply:
Thank you for supporting. It's OK